config.toml

```diff
@@ -12,7 +12,7 @@
 
 # This is the config for the server
 [server]
-	address	= "localhost"
+	address	= "0.0.0.0"
```

Comment:
tbh, I think this should be a config.toml.example file, and then we can copy our own into place properly